src/HDF5.jl

```diff
@@ -1153,15 +1163,29 @@ dataspace(n::Void) = HDF5Dataspace(h5s_create(H5S_NULL))
 dataspace(sz::Dims; max_dims::Union{Dims, Tuple{}}=()) = _dataspace(sz, max_dims)
 dataspace(sz1::Int, sz2::Int, sz3::Int...; max_dims::Union{Dims, Tuple{}}=()) = _dataspace(tuple(sz1, sz2, sz3...), max_dims)
 
-# Get the array dimensions from a dataspace or a dataset
-# Returns both dims and maxdims
+
 get_dims(dspace::HDF5Dataspace) = h5s_get_simple_extent_dims(dspace.id)
+"""get_dims(dset::HDF5Dataset)
+Get the array dimensions from a dataset
+Returns a tuple of dims and maxdims
+"""
 get_dims(dset::HDF5Dataset) = get_dims(dataspace(checkvalid(dset)))
 
-# change the current dimensions of a dataset, new_dims fit in max_dims = get_dims(dset)[2]
-# reduction is possible, and leads to loss of truncated data
+"""set_dims!(dset::HDF5Dataset, new_dims::Dims)
+ change the current dimensions of a dataset, new_dims fit in max_dims = get_dims(dset)[2]
+reduction is possible, and leads to loss of truncated data
+"""
```

Comment:
`Change the current dimensions of a dataset to `new_dims` to fit in `max_dims = get_dims(dset)[2]` . Reduction is possible and leads to loss of truncated data.`